recipe/meta.yaml

```diff
@@ -65,12 +65,17 @@ requirements:
     - fontconfig
     - libcurl
     - gettext  # [osx]
+    - nss >=3.19
+    - boost >=1.58.0
+    - lcms2
+    - poppler-data  # [unix]
+    - qt >=5.9
```

Comment:
Do we need Qt and the other dependencies? I thought they were for building an optional GUI/client that we don't ship.